File: backend/app/Models/Matricula.php
<?php


namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Matricula extends Model {
    protected $fillable = [
        'matricula', 'tipo', 'localizador', 'estado'
    ];
    protected $table="matriculas";

    public $timestamps = false;
}

File: frontend/app/src/main/java/com/example/frontend/controller/models/Matricula.kt
package com.example.frontend.controller.models

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "matriculas")
class Matricula
    (@PrimaryKey(autoGenerate = true)
    val id: Int,
    val matricula: String,
    val tipo: String,
     val localizador: Int,
    val estado: String)
